server/read-page.js

```diff
@@ -12,11 +12,15 @@ async function readPage (path) {
     return cache[f]
   }
 
-  const source = await fs.readFile(f, 'utf8')
-  const { component } = JSON.parse(source)
+  let content = await fs.readFile(f, 'utf8')
```

Comment:
@msand Would you mind removing all the changes done to this page.
We always need to read `.json` files so if there's no `.json` file, that's an error.